index.html

```diff
@@ -0,0 +1,228 @@
+
+
+        </section>
+
+        <section id="featured">
+            <h3 class="featuredSubTitle">Featured Speakers</h3>
+            <img class="underline" src="img/text/underline.svg" alt="underline">
+
+            <div  class="gridContainer">
+           
+            </div>
+
+            <button id="more">MORE <img id="orangeArrow" src="img/text/orangeArrow.svg" alt="Orange Arrow"></button>
+        </section>
+
+        <section id="partner">
+            <h3 class="featuredSubTitle" id="partnerTitle">Partner</h3>
+            <img class="underline" src="img/text/underline.svg" alt="underline">
+
+            <ul id="partnerListContainer">
+                <li class="partnerList"><img src="img/logos/mozilla.svg" alt="Mozilla"></li>
+                <li class="partnerList"><img src="img/logos/google.svg" alt="Google"></li>
+                <li class="partnerList"><img src="img/logos/naver.svg" alt="Naver"></li>
+                <li class="partnerList"><img src="img/logos/daumkakao.svg" alt="Daumkakao"></li>
+                <li class="partnerList"><img src="img/logos/airbnb.svg" alt="AirBNB"></li>
+            </ul>
+
+        </section>
+    </main>
+
+    <footer id="footer">
+        <img id="footerLogo" src="img/logos/creativecommons.svg" alt="Creative Commons Korea">
+        <div id="footerText">
+            <h4 id="footerTitle">2015 Creative Commons Korea. Some Rights Reserved</h4>
+            <p id="footerP">달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다. 자세한 이용기능 내용과 허락조건은
+                http://creativecommons.org/licences/by/2.0/kr 를 보세요.</p>
+        </div>
+    </footer>
+
+
+
+    <script src="/script.js"></script>
+</body>
+
+</html>
+
+     <!-- <div class="div1 cardContainer">
+                    <div class="cardPictureContainer"><img class="cardPicture" src="img/pictures/dude1.svg" alt=""></div>
+                    <div>
+                        <h2 class="cardTitle">Yochai Benkler</h2>
+                        <h3 class="cardsSubtitle">Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h3>
+                        <img class="cardUnderline" src="img/text/grayunderline.svg" alt="underline">
+                        <p class="cardsP">Benkler studies commons-based peer production, and published his seminal book, The
+                            Wealth of Networks in 2006</p>
+        
+                    </div>
+                </div>
+                <div class="div2 cardContainer">
+                    <div class="cardPictureContainer"><img class="cardPicture" src="img/pictures/dude1.svg" alt=""></div>
+                    <div>
+                        <h2 class="cardTitle">Yochai Benkler</h2>
+                        <h3 class="cardsSubtitle">Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h3>
+                        <img class="cardUnderline" src="img/text/grayunderline.svg" alt="underline">
```

Comment:
- [ ] It is not good practice to have this much commented code in your codebase. Kindly delete it to keep your work clean and professional 👍

Reply:
Ok, done!